books/free-programming-books-ja.md

```diff
@@ -685,26 +685,26 @@
 ### TypeScript
 
 * [TypeScript Deep Dive 日本語版](https://typescript-jp.gitbook.io/deep-dive/) - basarat, yohamta(翻訳)
-* [TypeScriptの為のクリーンコード](https://msakamaki.github.io/clean-code-typescript/) - labs42io, 酒巻 瑞穂(翻訳)
+* [TypeScriptの為のクリーンコード](https://msakamaki.github.io/clean-code-typescript) - labs42io, 酒巻 瑞穂(翻訳)
 
 
 #### Angular
 
-> :information_source: See also &#8230; [AngularJS](#angularjs)
+> :information_source: 関連項目 - [AngularJS](#angularjs)
```

Comment:
Ellipsis is part of crosslinks proposal #5535 

```suggestion
> :information_source: 関連項目 &#8230; [AngularJS](#angularjs)
```

Reply:
Comment welcome of crosslinks syntax. this gives us another option.